dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/trackedentity/TrackedEntityServiceTest.java

```diff
@@ -1794,4 +1839,15 @@ private void makeProgramMetadataInaccessible(Program program) {
     program.getSharing().setPublicAccess(AccessStringHelper.DEFAULT);
     manager.update(program);
   }
+
+  private TrackedEntityOperationParams createOperationParams(
```

Comment:
I would not create this helper as:
* all other tests lay out how the service is called which often shows what the test tests
* and it's not clear what operation params you are creating here 🤷🏻 

The setup of the data is much more in the way of test readability IMHO than the lines of calls to the params builder.